src/applications/combined-debt-portal/debt-letters/containers/DebtDetails.jsx

```diff
@@ -37,9 +36,9 @@ const DebtDetails = () => {
     ?.filter(history => approvedLetterCodes.includes(history.letterCode))
     .reverse();
   const hasFilteredHistory = filteredHistory && filteredHistory.length > 0;
-  const hasPaymentHistory =
```

Comment:
This is a hack to get rid of an error.

@Dr-Pongo, can you please help me with fixing this properly? Thanks!